HW04.Rmd

```diff
@@ -17,10 +19,20 @@ output: github_document
 
 The gun_deaths.csv is provided in the repository. 
 ```{r load-data, message=FALSE, warning=FALSE}
-library(tidyverse)    # load tidyverse packages, including ggplot2
+# load tidyverse packages, including ggplot2
+library(tidyverse) 
+library(ggplot2)
+library(knitr)
+library (devtools)
+
+#set the theme for all plots
+theme_set(theme_classic())

```

Comment:
I like that you set the theme at the top for all the plots so it is consistent throughout, really good thinking ahead!